service/application/src/main/java/com/solace/maas/ep/event/management/agent/processor/RouteFailedProcessorImpl.java

```diff
@@ -0,0 +1,29 @@
+package com.solace.maas.ep.event.management.agent.processor;
```

Comment:
New processor to set the scan status for a scan type. Before it always stayed Initiating in the DB. Now we set it to failed so the command line runner knows that the command failed.